content/reference/storage/reading-key-value-pairs.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Reading key-value pairs'
+weight: 80
+---
+
+To get the value for a given key, you can call the `get` method on any
+namespace you've bound to your script:
+
+`NAMESPACE.get(key, [type])`
+
+The method returns a promise you can `await` to get the value. If the key
+is not found, the promise will resolve with the literal value `null`.
+
+The `type` parameter is optional, and can be any of:
+
+- `"text"` (default)
+- `"json"`
+- `"arrayBuffer"`
+- `"stream"`
+
+We'll talk more about these types below.
+
+Here's an example of reading a key from within a Worker:
+
+{{<highlight javascript>}}
```

Comment:
While this works in Hugo, we also support GFM syntax, which I think will probably port better to whatever framework we move to next year. Can you update with that syntax?